ansible/inventories/devnet-7/group_vars/lodestar.yaml

```diff
@@ -36,9 +36,8 @@ lodestar_container_volumes:
 lodestar_container_command_extra_args:
   - --paramsFile=/network-config/config.yaml
   - --genesisStateFile=/network-config/genesis.ssz
-  - --rest.namespace=*
-  #- --logLevel=debug
-  - --bootnodes={{ ethereum_cl_bootnodes | join(',') }}
+  - --rest.namespace="*"
+  - --bootnodes=/data/ethereum-network-config/boot_enr.txt
```

Comment:
sure they support reading from the file?

Reply:
It works for lodestar and testing for all other clients now.

Reply:
Teku: https://github.com/Consensys/teku/issues/7334

Reply:
LH supports it using an union between CLI arg and testnet-dir with boot_enr.yaml https://discord.com/channels/605577013327167508/605577013331361793/1126464572849659926

Reply:
I'd suggest holding off on the change until they all support it. Its a minor difference for now and having different clients behave differently would cause headaches when things go wrong.

Reply:
In order to be able to reuse the commands from this in the kubernetes deployment, this change is needed, otherwise its going to lead to issues. 
As we don't generally use teku for explorer/tooling, I think its a fine change to have.

Reply:
wait reuse the commands from kubernetes? that sounds messy, are you trying to enforce the same commands from ansible onto kubernetes nodes?

Reply:
Yes. 
Otherwise we need to have a new list of commands for kube which seems redundant to me.

Reply:
Well one is meant to be the RPC and handle outgoing traffic and other might need to be for debugging reasons, I can see there being real need to separate the two. I guess we can go with combined and then use hostvars for the deviations, and decide later on if its too much friction. 

Did we ever test if you can add `if` jinja blocks into the vars here and if its respected by the role? That'd kinda solve the problem too.